tests/unit/config/test_configinit.py

```diff
@@ -177,6 +177,7 @@ def test_autoconfig_yml(self, init_patch, config_tmpdir, caplog, args,
             assert config.instance._values == {}
         else:
             assert config.instance._values == {'colors.hints.fg': 'magenta'}
+        # pylint: enable=too-many-locals,too-many-branches
```

Comment:
No need for this, as pylint already only turns things off for this function and it's needed for the entire function.